src/components/SearchBar.jsx

```diff
@@ -1,20 +1,55 @@
 import React from "react";
+import axios from "axios";
 
 import Form from "react-bootstrap/Form";
 import InputGroup from "react-bootstrap/InputGroup";
 import Button from "react-bootstrap/Button";
 
 export default class SearchBar extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      searchTerm: "",
+    };
+  }
+
+  handleSearchInput = ({ target: { value: searchTerm } }) => {
+    this.setState({ searchTerm });
+  };
+
+  handleSubmit = async (event) => {
+    event.preventDefault();
+    const { setCurrentCity } = this.props;
+    const apiURL = `https://us1.locationiq.com/v1/search?key=${
+      import.meta.env.VITE_LOCATIONIQ_API_KEY
```

Comment:
Probably just a personal style thing, but I try to keep variables and their containers on the same line, i.e. ${........} and then try and do consistent indentation, if I'm gonna break a long string down.